working_notebook-TO_COMPLETE_Anna.ipynb

```diff
@@ -815,19 +815,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.                    if number &gt; 20:
</pre><p>now that you know the <code>min</code>  value, this line could be simply replaced by </p><p><code>number = min(number, 20)</code>   don't you think? :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/programming-for-data-science/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>